### PR TITLE
fix: mason_logger v0.1.0-dev.9 compat

### LINKED ISSUE #408: fix: cannot activate very_good_cli after mason_logger-v0.1.0-dev.9 update
**Description**
Trying to activate `very_good_cli` fails with following error:

```sh
$ dart pub global activate very_good_cli
Resolving dependencies...
+ archive 3.3.0
+ args 2.3.1
+ async 2.9.0
+ checked_yaml 2.0.1
+ collection 1.16.0
+ crypto 3.0.2
+ file 6.1.2
+ glob 2.0.2
+ http 0.13.4
+ http_parser 4.0.1
+ json_annotation 4.5.0
+ lcov_parser 0.1.2
+ mason 0.1.0-dev.17 (0.1.0-dev.24 available)
+ mason_logger 0.1.0-dev.9
+ meta 1.8.0
+ mustache_template 2.0.0
+ path 1.8.2
+ platform 3.1.0
+ process 4.2.4
+ pub_semver 2.1.1
+ pub_updater 0.2.2
+ pubspec_parse 1.2.0
+ recase 4.0.0
+ source_span 1.9.0
+ stack_trace 1.10.0
+ string_scanner 1.1.1
+ term_glyph 1.2.0
+ typed_data 1.3.1
+ universal_io 2.0.4
+ usage 4.0.2
+ very_good_analysis 2.4.0 (3.0.0 available)
+ very_good_cli 0.7.3
+ very_good_test_runner 0.1.2
+ yaml 3.1.1
Building package executables...
Failed to build very_good_cli:very_good:
AppData/Local/Pub/Cache/hosted/pub.dartlang.org/very_good_cli-0.7.3/lib/src/commands/packages.dart:68:29: Error: The argument type 'Progress Function(String)' can't be assigned to the parameter type 'void Function([String?]) Function(String)?'.
 - 'Progress' is from 'package:mason_logger/src/progress.dart' ('AppData/Local/Pub/Cache/hosted/pub.dartlang.org/mason_logger-0.1.0-dev.9/lib/src/progress.dart').
          progress: _logger.progress,
                            ^
AppData/Local/Pub/Cache/hosted/pub.dartlang.org/very_good_cli-0.7.3/lib/src/commands/test/test.dart:137:29: Error: The argument type 'Progress Function(String)' can't be assigned to the parameter type 'void Function([String?]) Function(String)?'.
 - 'Progress' is from 'package:mason_logger/src/progress.dart' ('AppData/Local/Pub/Cache/hosted/pub.dartlang.org/mason_logger-0.1.0-dev.9/lib/src/progress.dart').
          progress: _logger.progress,
                            ^
```

**Steps To Reproduce**

1. Run `dart pub global activate very_good_cli`
1. See error

**Expected Behavior**
We can activate and use very_good_cli without any errors.

**Additional Context**
This error occurs due to progress api enhancements in [mason_logger-v0.1.0-dev.9](https://github.com/felangel/mason/releases/tag/mason_logger-v0.1.0-dev.9).

### PR DESCRIPTION
**Pull request by felangel**
## Description

- fix: mason_logger v0.1.0-dev.9 compat (closes #408)

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [X] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/src/commands/packages.dart [hunk 1]
```diff
@@ -65,7 +65,7 @@ class PackagesGetCommand extends Command<int> {
         await Flutter.packagesGet(
           cwd: targetPath,
           recursive: recursive,
-          progress: _logger.progress,
+          progress: (value) => _logger.progress(value),
```
Comment (wolfenrain) [inline lib/src/commands/packages.dart:68]: How does this complete?

**Review by wolfenrain — APPROVED** [review]

File: lib/src/commands/packages.dart (hunk 1 above)
Comment (felangel) [reply lib/src/commands/packages.dart]: oops good catch! just updated to proxy to `complete`

Comment (wolfenrain) [reply lib/src/commands/packages.dart]: I think in the long run we might want to provide the whole progress object? As the tests can fail and we then can show that better? For now, LGTM!

Comment (felangel) [reply lib/src/commands/packages.dart]: yeah I'm going to close this for now in favor of https://github.com/VeryGoodOpenSource/very_good_cli/pull/411 and we can properly upgrade this week

**Pull request closed without merging**